pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -2089,3 +2114,55 @@ func getRouteHostname(obj interface{}) (host string) {
 	}
 	return host
 }
+
+func checkAndCreateAdminSecret(service kubernetes.PlatformService, namespace string, secretName string) (error, string, string) {
+	customerSecret, err := getSecret(service, namespace, secretName)
+
+	// The secret passed by the user doesn't exist we create one with default values
+	if err != nil {
+		username := constants.DefaultAdminUser
+		password := string(shared.GeneratePassword(8))
```

Comment:
@spolti is it ok use a generated password or we want to use the default password like in the trial env ?